spec/column_spec.rb

```diff
@@ -37,7 +37,7 @@
     end
 
     it 'accepts bold' do
-      expect(CommandLineReporter::Column.new('asdf', :bold => true).bold).to be_true
+      expect(CommandLineReporter::Column.new('asdf', :bold => true).bold).to be true
```

Comment:
Line is too long. [84/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.